src/main/kotlin/com/ulu/Application.kt

```diff
@@ -51,8 +51,8 @@ class TestDataCreator(private val dbService: DatabaseService, private val userDa
             )
         val user2 =
             UserData(
-                name = "Paul",
-                email = "pauling@gmail.com",
+                name = "Jimmy",
```

Comment:
Why?

Reply:
Because it is funny, and because @Requires(notEnv = ["prod", "test"]) was not initially set to exclude "test".
This caused tests to failed since multiple users with the same name where created when running the applications and tests at the same time.